.github/workflows/ci.yml

```diff
@@ -178,7 +174,15 @@ jobs:
             ~/.cargo/git
             ~/.cargo/registry
             ./target
-          key: cache4-${{ matrix.os }}-${{ matrix.kind }}-${{ hashFiles('Cargo.lock') }}
+          key:
+            ${{ matrix.os }}-${{ matrix.kind }}-${{ hashFiles('Cargo.lock') }}
+          restore-keys: |
+            ${{ matrix.os }}-${{ matrix.kind }}-
```

Comment:
no restore key?

Reply:
I don't understand... You want it gone?

Reply:
Oops - no I read the diff wrong. Nevermind.